tools/medenv/med_environ.xml

```diff
@@ -0,0 +1,129 @@
+<tool id="iabiodiv_smartbiodiv_med_environ" name="Adds the environment variables to a dataset" version="0.0.1">
+    <description>
+        From the input dataset, it will request copernicus for environment variables.
+    </description>
+
+	<requirements>
+        <requirement type="package" version="2.1.3">pandas</requirement>
+        <requirement type="package" version="4.9.3">lxml</requirement>
+        <requirement type="package" version="1.0.4">medenv</requirement>
+	</requirements>
+
+    <command>
```

Comment:
feel free to add https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-creator-person